Inference/c++/ernie/cmake/FindFluid.cmake

```diff
@@ -84,8 +84,7 @@ endfunction()
 third_party_include(glog glog/logging.h ${THIRD_PARTY_ROOT}/install/glog/include)
 third_party_include(protobuf google/protobuf/message.h ${THIRD_PARTY_ROOT}/install/protobuf/include)
 third_party_include(gflags gflags/gflags.h ${THIRD_PARTY_ROOT}/install/gflags/include)
-third_party_include(eigen unsupported/Eigen/CXX11/Tensor ${THIRD_PARTY_ROOT}/eigen3)
-third_party_include(boost boost/config.hpp ${THIRD_PARTY_ROOT}/boost)
+third_party_include(cryptopp cryptopp/cryptlib.h ${THIRD_PARTY_ROOT}/install/cryptopp/include)
```

Comment:
Why eigen and boost were removed?

Reply:
Hi eigen and boost is not in that location anymore. 
I build Paddle inference lib .locally as well as download from https://www.paddlepaddle.org.cn/documentation/docs/zh/guides/05_inference_deployment/inference/build_and_install_lib_cn.html these two folders are not in inference lib anymore. 
I also confirmed with Juncai he said they are not in inference lib anymore.
Also because `inference.cc` did not use `config.hpp` or boost header so I think deleting them should be fine